tests/xbot/common/subsystems/pose/TestPoseSubsystem.java

```diff
@@ -30,4 +35,10 @@ protected double getLeftDriveDistance() {
     protected double getRightDriveDistance() {
         return right.getPosition();
     }
+    
+    public void setDistanceTraveled(double left, double right) {
```

Comment:
this isn't exactly setting the 'distanceTraveled' directly though right? Might be about setDriveEncoderDistances ?